extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -45,17 +77,20 @@ describes.realWin(
       config = defaultInlineConfig
     ) {
       pageElement.appendChild(
-        <amp-story-shopping-config layout="nodisplay" src={src}>
+        <amp-story-shopping-attachment layout="nodisplay" src={src}>
           <script type="application/json">{JSON.stringify(config)}</script>
-        </amp-story-shopping-config>
+        </amp-story-shopping-attachment>
+      );
+
+      return getShoppingConfig(
+        pageElement.getElementsByTagName('amp-story-shopping-attachment')[0]
```

Comment:
Don't query if you already have a reference. Set `<amp-story-shopping-attachment>` to a variable named `element` and pass that to `getShoppingConfig`